client/js/widgets/button.js

```diff
@@ -83,8 +93,6 @@ export class Button extends Widget {
         continue;
       }
 
-
```

Comment:
I don't really care but this seems like a removal for no reason.

Reply:
single blank line is simply more consistent, same reason as for following other conventions